Month3/Project1/Project1/ArrayVectors.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+#define _USE_MATH_DEFINES
+#include <iostream>
+#include <iomanip>
+#include <cmath>
+
+class FShape
```

Comment:
This header file is somewhat confusing, as it is the header file of the entire program. However, it includes the class FShape, which is in no way related to the header name, ArrayVectors. I would expect that when I import ArrayVectors I would get a header file for a program that implements those, not the class FShape. Furthermore, if I want to find the class FShape, I would probably have to search for it in the file directory. 

Once again, I know this class is small, but I think it's a good practice to follow.

Reply:
Definitely important. It makes no sense to have the FShape class declared and defined in files named ArrayVectors.